configuration_files/hsr.lua

```diff
@@ -21,7 +21,7 @@ options = {
   published_frame = "odom",
   odom_frame = "odom",
   provide_odom_frame = false,
-  use_odometry_data = true,
+  use_odometry_data = false,
```

Comment:
Huh. We do have an IMU, I guess we just forgot to include it in the bag?  If that's the case, should this actually be changed in the configuration?